### PR TITLE
fix(Avatar): improve accessibility

### PR DESCRIPTION
**Pull request by langz**
Adds label for screen readers to Avatar.Group, and uses the full string passed to Avatar as label for screen readers.

### CODE REVIEW

**Review by tujoworker — APPROVED** [review]
Omg! So fast, beautiful 😍 and well done!

File: packages/dnb-eufemia/src/components/avatar/AvatarGroup.tsx [hunk 1]
```diff
@@ -11,6 +11,12 @@ import { ISpacingProps } from '../../shared/interfaces'
 import { extendPropsWithContext } from '../../shared/component-helper'
 
 export interface AvatarGroupProps {
+  /**
+   * Aria label to describe the tag group
```
Comment (thaytharma) [inline packages/dnb-eufemia/src/components/avatar/AvatarGroup.tsx:15]:
```suggestion
   * Label to describe the avatar group
```

File: packages/dnb-eufemia/src/components/avatar/__tests__/Avatar.test.tsx [hunk 2]
```diff
@@ -69,10 +120,32 @@ describe('Avatar', () => {
     expect(image.getAttribute('height')).toBe(img_height)
   })
 
+  it('warns when Tag is used without a Tag.Group as parent component', () => {
```
Comment (thaytharma) [inline packages/dnb-eufemia/src/components/avatar/__tests__/Avatar.test.tsx:123]:
```suggestion
  it('warns when Avatar is used without a Avatar.Group as parent component', () => {
```

File: packages/dnb-eufemia/src/components/avatar/__tests__/Avatar.test.tsx [hunk 3]
```diff
@@ -69,10 +120,32 @@ describe('Avatar', () => {
...
+    process.env.NODE_ENV = 'development'
+    global.console.log = jest.fn()
+    mount(<Avatar />)
+    expect(global.console.log).toBeCalled()
+  })
+
   describe('AvatarGroup', () => {
+    it('renders the label', () => {
+      const label = 'tags'
```
Comment (thaytharma) [inline packages/dnb-eufemia/src/components/avatar/__tests__/Avatar.test.tsx:132]:
```suggestion
      const label = 'avatars'
```

**Review by thaytharma — APPROVED** [review]
Looks good! Added a few suggested corrections 😊

File: packages/dnb-eufemia/src/components/avatar/__tests__/Avatar.test.tsx (hunk 2 above)
Comment (langz) [reply packages/dnb-eufemia/src/components/avatar/__tests__/Avatar.test.tsx:123]: 😂 🙏

File: packages/dnb-eufemia/src/components/avatar/__tests__/Avatar.test.tsx (hunk 3 above)
Comment (langz) [reply packages/dnb-eufemia/src/components/avatar/__tests__/Avatar.test.tsx:132]: Danke danke

Comment (langz) [reply packages/dnb-eufemia/src/components/avatar/__tests__/Avatar.test.tsx:132]: You can say that it "gikk litt fort i svingene" 😂 
And I had such great examples from your code in tags 🙏

**Comment by langz** [comment]
> Looks good! Added a few suggested corrections 😊

You can say that it "gikk litt fort i svingene" 😂 
And I had such great examples from your code in tags 🙏

**Comment by thaytharma** [comment]
> > Looks good! Added a few suggested corrections 😊
> 
> You can say that it "gikk litt fort i svingene" 😂 And I had such great examples from your code in tags 🙏

Hahah, kjenner meg igjen.. 
Great work anyways. 😂

**Pull request merged**